stable/gitlab-ce/templates/deployment.yaml

```diff
@@ -0,0 +1,118 @@
+{{- if default "" .Values.externalUrl }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
```

Comment:
Not without a distributed filesystem and some other things. That'll definitely be one of those cases where more advanced users will fork and customize to their setup.